src/test/pages/customersPage.page.ts

```diff
@@ -35,24 +39,24 @@ export class CustomerPage extends MembersPage {
     }
 
     public async addInvalidEmail() {
-
+      await this.emailInput.fill("  ")
```

Comment:
So validating invalid parameters will always be performed by passing whitespace?
I would clarify what values are invalid and would pass every time different invalid parameter.
Another way will be to pass invalid value from feature file. 
Please update it.

Reply:
the only invalid  character that is invalidated by the email box is